.github/workflows/release.yaml

```diff
@@ -110,6 +118,15 @@ jobs:
           file: config/.release/install.yaml
           asset_name: install.yaml
           tag: ${{ github.ref }}
+      
+      - name: Push yaml manifest into an OCI registry
+        run: |
+            cosign login --username ${GITHUB_ACTOR} --password ${{ secrets.CR_PAT }} ghcr.io
+            flux push artifact oci://ghcr.io/${{ github.repository_owner }}/manifests/kyverno:$(git rev-parse --short HEAD) \
+            --path="install.yaml" \
+            --source="$(git config --get remote.origin.url)" \
+            --revision="$(git branch --show-current)/$(git rev-parse HEAD)"
+            cosign sign ghcr.io/${{ github.repository_owner }}/manifests/kyverno:$(git rev-parse --short HEAD)
```

Comment:
Cosign keyless requires a couple of things:

Add permissions to this workflow:

```yaml
permissions:
  contents: write # needed to write releases
  id-token: write # needed for keyless signing
  packages: write # needed for ghcr access
```

Add `COSIGN_EXPERIMENTAL=1` to the sign command